api/src/pcapi/routes/backoffice/finance/blueprint.py

```diff
@@ -307,7 +311,7 @@ def create_individual_booking_incident() -> utils.BackofficeResponse:
                 bookingId=booking.id,
                 incidentId=incident.id,
                 beneficiaryId=booking.userId,
-                newTotalAmount=-booking.pricing.amount if len(bookings) > 1 else int(form.total_amount.data * 100),
+                newTotalAmount=(booking.total_amount if len(bookings) > 1 else form.total_amount.data) * 100,
```

Comment:
[détail] Mieux vaut utiliser `finance_utils.to_eurocents()` qui fait (à peu près) la même chose mais renvoie un entier. Ici ça ne serait pas un problème de simplement multiplier par 100, mais c'est une bonne habitude d'utiliser cette fonction, car il y a des endroits où multiplier par 100 ne fonctionnerait pas (cf. le corps de la fonction `to_eurocents()` et ses tests).

```suggestion
                newTotalAmount=finance_utils.to_eurocents(booking.total_amount if len(bookings) > 1 else form.total_amount.data),
```